week-3/todo-list/index.html

```diff
@@ -3,25 +3,36 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Title here</title>
+    <title>Todo List</title>
+    <link
+      rel="stylesheet"
+      href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
     <link rel="stylesheet" href="style.css" />
+    <script src="script.js" defer></script>
   </head>
   <body>
-    <form>
-      <div>
-        <input type="text" placeholder="New todo..." />
-      </div>
-      <div>
-        <button type="submit">Add Todo</button>
-        <button
-          type="button"
-          id="remove-all-completed"
-          class="btn btn-primary mb-3"
-        >
-          Remove all completed
-        </button>
-      </div>
-    </form>
-    <script src="script.js"></script>
+    <container class="content">
+      <section class="list-group">
+        <ul id="todo-list" ></ul>
+      </section>
+      <section>
+        <form>
+          <div>
+            <input id="todo-input" type="text" placeholder="New todo..." />
+          </div>
+          <div>
+            <button id="add-todo" type="submit">Add Todo</button>
+            <button
+              type="button"
+              id="remove-all-completed"
+              class="btn btn-primary mb-3"
```

Comment:
Doesn't look like you're using these class names in the CSS. They look like classes from Bootstrap CSS. When copy pasting always make sure you tidy up the code so that you can be certain to understand why it works :)